xss_utils/templatetags/django_markup.py

```diff
@@ -0,0 +1,32 @@
+"""
+Template tags and helper functions for escaping script tag.
+"""
+from django import template
+from django.utils import html
+
+register = template.Library()
+
+
+@register.filter(name="htmlescape")
+def htmlescape(value):
```

Comment:
This one seems like it is built into Django Templates.  Can you show me why we need this and how it is used?

Reply:
@robrap It is used here https://github.com/edx/credentials/blob/master/credentials/templates/credentials/programs/base.html#L115
It was already there in credential code base. I should have previously used built in construct instead of creating a new filter as suggested in wiki, but didn't knew about it and i think the document was created after initial code merge in credential.    
https://openedx.atlassian.net/wiki/spaces/SEC/pages/330760578/Preventing+XSS+in+Django+Templates
```
{% trans "somestring" as tmsg %}{{ tmsg | force_escape }}
```
So either we should remove it from credentials right now or let it keep on using htmlscape filter for the time being and incrementally remove from both repos, after this work is completed. What you suggest?

Reply:
I would remove `htmlescape` from this repository.

When credentials is updated to use this new repo, you can either:
1. Remove all usages of `htmlescape` and replace with `force_escape` if that works, or
2. Leave `htmlescape` in credentials and continue to use it there only.  If this is done, please update with a comment mentioning what is in the wiki (a.k.a. that it is deprecated and that you should be using `force_escape` instead.

Doing the first option would be cleanest if it works for obvious reasons.